packages/next/src/server/config-shared.ts

```diff
@@ -157,8 +172,22 @@ export interface ExperimentalConfig {
 
   webVitalsAttribution?: Array<typeof WEB_VITALS[number]>
 
-  // webpack loaders to use when running turbopack
-  turbopackLoaders?: Record<string, string | string[]>
+  /**
+   * (`next --turbo` only) A mapping of aliased imports to modules to load in their place.
+   *
+   * @see [Resolve Alias](https://nextjs.org/docs/api-reference/next.config.js/resolve-alias)
+   */
+  resolveAlias?: Record<
```

Comment:
This should probably be nested under `turbo` or prefixed with it so users don't think it applies without.